mailpoet/tests/acceptance/Settings/ConfirmConfirmationPageCest.php

```diff
@@ -12,7 +12,7 @@ public function confirmDefaultConfirmationPage(\AcceptanceTester $i) {
     $i->waitForText('MailPoet Page');
     $i->click('[data-automation-id="preview_page_link"]');
     $i->switchToNextTab();
-    $i->waitForText('You have subscribed to: demo 1, demo 2');
+    $i->see('You have subscribed to');
```

Comment:
Do you think it'd make sense to check for the site name in this assertion as well?

Reply:
Not necessarily. I think this is okay as well.
If we can see the string on the page, the test case should be allowed to pass